addon.py

```diff
@@ -31,8 +31,9 @@ def __init__(self, addon_url, addon_handle, addon_args):
         self.username = addon.getSetting("username")
         self.password = addon.getSetting("password")
 
-        self.random_count = addon.getSetting("random_count")
-        self.bitrate = addon.getSetting("bitrate")
+        self.random_count = int(addon.getSetting("random_count"))
```

Comment:
Not sure what will happen if `addon.getSetting("random_count")` returns a non-integer. Maybe wrap it in something like

```python
try:
    self.bitrate = int(addon.getSetting("random_count"))
except ValueError:
    self.bitrate = 0
```

Same applies for other calls below.